sdks/python/scripts/run_snapshot_publish.sh

```diff
@@ -30,8 +30,10 @@ DEP_SNAPSHOT_FILE_NAME="beam-py-requirements-$time.txt"
 # and located under Gradle build directory.
 cd $WORKSPACE/sdks/python/build
 
-# Rename the file to be apache-beam-{VERSION}-{datetime}.tar.gz
-for file in "apache-beam-$VERSION*.tar.gz"; do
+# Rename the file to be apache_beam-{VERSION}-{datetime}.tar.gz
+# Notice that the distribution name of beam has been normalized as "apache_beam"
+# since setuptools 69.3.0.
+for file in "apache_beam-$VERSION*.tar.gz"; do
```

Comment:
shall we use `apache?beam-$VERSION*` so both are supported?

Reply:
Good point, I thought we pinned the setuptools, but we do not.

Reply:
Done.